src/source/mod.rs

```diff
@@ -3,6 +3,8 @@
 //! In cernan a `Source` is a place where all `metric::Event` come from, feeding
 //! down into the source's forwards for further processing. Statsd is a source
 //! that creates `Telemetry`, `FileServer` is a source that creates `LogLine`s.
+extern crate mio;
```

Comment:
Marker to remove redundant extern create.

Reply:
Fixed.